src/app/components/events/create-event/create-event.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { EventRequest } from "../../../entity/EventRequest";
+import { EventTemplate } from "../../../entity/EventTemplate";
+import { CompleteBetType } from "../../../entity/CompleteBetType";
+import { BetTemplateType } from "../../../entity/BetTemplateType";
+import { UserProfile } from "../../../entity/UserProfile";
+import { Status } from "../../../entity/Status";
+import {EventService} from "../../../service/event.service";
+import {EventTemplateService} from "../../../service/event-template.service";
+import {HttpClient} from "@angular/common/http";
+
+@Component({
+  selector: 'app-create-event',
+  templateUrl: './create-event.component.html',
+  styleUrls: ['./create-event.component.scss']
+})
+export class CreateEventComponent implements OnInit {
+  formData: EventRequest = new EventRequest();
+  eventTemplates: EventTemplate[] = [];
+  userGroupsControl = new FormControl();
+  userProfilesControl = new FormControl();
+
+  userGroupsList: string[] = ['Group1', 'Group2', 'Group3'];
+  userProfilesList: UserProfile[] = [
+    { userId: 1, keycloakId: 'key1', username: 'User1', profilePicture: 1, description: 'Description1', bets: [], coins: 50 },
+    { userId: 2, keycloakId: 'key2', username: 'User2', profilePicture: 2, description: 'Description2', bets: [], coins: 50 },
+  ];
+  statusOptions: Status[] = [Status.DRAFT, Status.OPEN, Status.CLOSED];
+
+  constructor(private eventTemplateService: EventTemplateService ,
+              private eventService :EventService ,
+              private http: HttpClient) {}
+
+  ngOnInit() {
+    // Call the method from the service to fetch event templates on component initialization
+    this.eventTemplateService.getData().subscribe((data: { entity: EventTemplate[]; }) => {
+      this.eventTemplates = data.entity;
+    });
+    this.http.get<UserProfile[]>('http://localhost:8080/api/user-profiles').subscribe(
```

Comment:
Don't directly inject http client into component, please use method from user profile service or create a method these in case it doesn't exist